src/functions/unreferencedReusableObject.ts

```diff
@@ -1,5 +1,6 @@
+import { isObject } from 'lodash';
```

Comment:
The docs talk about "Cherry-pick methods for smaller browserify/rollup/webpack bundles." where you `import isObject at from 'lodash/isObject';` instead. Is this something we should do too?

Reply:
We used to have similar imports back in the day, but changed them in all projects across the whole Stoplight organization.


@marbemac was typings an issue? IIRC that was it, but don't remind myself exactly now.
If I am not mistaken, we had to use that awkward import-require syntax, `import isObject = require('lodash/isObject');`

Reply:
It was because tree shaking was not working quite correctly - importing from more specific `lodash/{foo}` type target doesn't rely on tree shaking. We have since moved away from the specific imports though, and to the nicer `import { foo } from 'bar'` style syntax.

Reply:
I dunno what tree shaking means I’ll have a google. The only reason I pointed this out is I’ve seen mention that this mixer syntax creates much larger webpack builds due to something something can’t remember why. Might be a worthwhile optimisation, might be a microoptimization.

Reply:
Yeah it's not worth it - in order to make the `lodash/isObject` type syntax work, we'd have to use that syntax consistently everywhere, and hope that our dependencies do as well. Better to try to get tree shaking working, but even then it's not a huge difference in output bundle size.